data_steward/analytics/cdr_ops/ehr_snapshot_qc.py

```diff
@@ -0,0 +1,59 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + tags=["parameters"]
+project_id = ""
+old_ehr = ""
+new_ehr = ""
```

Comment:
Comments describing the expected values will be helpful the next time we run this notebook.